### PR TITLE
rust 1.55.0

### PR DESCRIPTION
**Pull request by Bo98**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/rust.rb [hunk 1]
```diff
@@ -2,24 +2,16 @@ class Rust < Formula
...
-          tag:      "0.55.0",
-          revision: "5ae8d74b3b2d58f32c8d357e5cfa04d430a70e0b"
-
-      # Work around crates.io timeouts with newer curl versions
-      # https://github.com/rust-lang/cargo/pull/9695
-      patch do
-        url "https://github.com/rust-lang/cargo/commit/27277d966b3cfa454d6dea7f724cb961c036251c.patch?full_index=1"
-        sha256 "55bf95b62dd38f372747922e5530544321601cf4c0e87e8174f440aa874f522d"
-      end
+          tag:      "0.56.0",
```
Comment (carlocab) [inline Formula/rust.rb:13]:
```suggestion
```
The tag doesn't exist yet, unfortunately.

Comment (Bo98) [reply Formula/rust.rb:13]: It's there now.

**Comment by Bo98** [comment]
Build takes about as long as the stale bot timeout

**Comment by Bo98** [comment]
Only one Eust-related failure I think (probably best to test fixing this after merge):

* theharvester - needs `orjson` update to at least 3.6.2.

Rest don't seem related to Rust update. As a quick summary of Intel:

* dotenv-linter - sha256 mismatch
* gimme-aws-creds - PyObjC bug relating to not recognising MacOSX11.sdk symlink
* gnuradio - can't find some qwt header
* homebank - sha256 mismatch
* msc-generator - not compatible with latest Graphviz
* pc6001vx - version header bug
* vice - can't find glew
* wasm-pack - sha256 mismatch

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**